benchmarks/TFLite/CMakeLists.txt

```diff
@@ -62,10 +62,22 @@ set(MOBILEBERT_FP32_MODULE
   NAME
     "MobileBertSquad"
   TAGS
-    "fp32"
+    "fp32,squad"
   SOURCE
-    # Mirror of https://tfhub.dev/tensorflow/lite-model/mobilebert/1/default/1
-    "https://storage.googleapis.com/iree-model-artifacts/mobilebertsquad.tflite"
+    "https://storage.googleapis.com/iree-model-artifacts/mobilebert-baseline-tf2-float.tflite"
```

Comment:
is this model different than https://storage.googleapis.com/iree-model-artifacts/mobilebertsquad.tflite in a material way that will affect the benchmark results?

Reply:
Please include where this model comes from. We just put it in the storage bucket for fast and reliable access, but it should generally be mirroring some model publicly available in a more canonical location. The addition of the tag here would be creating a new benchmark series and halting additions to our current one. It does seem like putting "squad" as a tag instead of in the model name is more consistent with what we do for imagenet below. I do think it should be instead rather than as well though. The tags appear alongside the model name, so both is redundant. We also need to think about whether we should be creating a new series for this (which in the current paradigm, changing either then name or tags will do). This is why I was opposed to using natural keys instead of artificial keys for these series FWIW.

Reply:
@julianwa The existing MobileBert model is TOCO converted using TF1.14, the new one is MLIR Converted using TF2.0. The architectures differ here and there with additional ops in one and not the other.

Waiting for the benchmark run to complete to see any differences in latency on mobile but on my desktop, here is a comparison using MLPerf:

Existing MobileBert:
90 percentile latency: 245.84 ms
Accuracy: 93.9833

New MobileBert:
90 percentile latency: 267.67 ms
Accuracy: 93.0667

Since TOCO is now deprecated, I think we should use the new one.

Reply:
+1 to what @GMNGeoffrey said. We need to keep the NAME and TAGS unchanged here to avoid creating a new benchmark series that is detached from the existing one. Also please match the new int8 version with it. 

> We also need to think about whether we should be creating a new series for this (which in the current paradigm, changing either then name or tags will do). 

I'd say we don't need to create a new series for this given this is only changing how it is implemented (TF1 -> TF2) and converted (TOCO -> MLIR). It's not a model architecture change (albeit a few op changes due to impl) like MobileNet V1 -> V2 -> V3, which each has its own paper. But @mariecwhite please feel free to correct me if I misunderstand it.

> This is why I was opposed to using natural keys instead of artificial keys for these series FWIW.

Hehe, I totally get your points. 😜  That's indeed an issue. But it's a trade off; it has the plus side that we can have very intuitive mapping (in URL and scripts, etc.) and it's simple. To use another mechanism as the "primary key" would mean we need to have maintain a mapping somewhere and handle previous/new version conversions of the same benchmark series. That's investment back then we don't have the resource to do. We can look into changing this later if it turns out to be truly annoying. But thus far we only really changed the name once. I'm happy to do another round to correct certain things after accumulating a few if necessary. :)

Reply:
> I'd say we don't need to create a new series for this given this is only changing how it is implemented (TF1 -> TF2) and converted (TOCO -> MLIR). It's not a model architecture change (albeit a few op changes due to impl) like MobileNet V1 -> V2 -> V3, which each has its own paper. But @mariecwhite please feel free to correct me if I misunderstand it.

Correct, it's the same architecture, just different implementation. I've removed the squad tags and now using "fp32" and "int8".

Reply:
> Please include where this model comes from. We just put it in the storage bucket for fast and reliable access, but it should generally be mirroring some model publicly available in a more canonical location.

These models were trained internally but can be reproduced externally (albeit with some effort). The architecture and implementation is public, and is trained on a public dataset.

Reply:
> Since TOCO is now deprecated, I think we should use the new one.
I agree with this. If the benchmark results are not that different for IREE, then I assume we'll just continue the benchmark history?

Reply:
I'm still a bit skeptical of this change, although my understanding is that finding a publicly-available quantized version is proving difficult. I have two concerns:

1. I'd prefer to not change the model within the same benchmark series. These are supposed to be measuring the same thing over time, so it feels wrong to change it. We could create a new series, but that has the same problem of the datasets being somewhat incomparable and comparing across datasets is difficult with our current system.
2. I prefer that we use public/standard models where possible. I think it's better to say "we took model x of tfhub and are benchmarking against it" than having to describe an internal-only process.

Now maybe the consideration that we couldn't find a quantized version of our current model overrides these concerns, but I do think we should careful here

Reply:
Note that the [benchmark results](https://gist.github.com/iree-github-actions-bot/fa179e45784e024cff3d7246287fb47c) only show changes on benchmarks not modified by this commit (so it's noise), which indicates that there was no change in performance with the change of the model file. I think that alleviates my concerns on that point somewhat.

Reply:
Yup. I think it's just implementation (TF1->TF2) and conversion (TOCO->MLIR) changes; the model architecture remains. So I think it's fine to use the same series.